client/src/pages/Edit.js

```diff
@@ -16,21 +17,13 @@ function Edit() {
 
   useEffect(() => {
     const editFoodId = async () => {
-      const response = await fetch(`http://localhost:5000/menu_items/${id}`); //change endpoint to hit 1 entry not all
-      const result = {
-        data: null,
-        error: null,
-      };
-
-      if (response.ok) {
-        result.data = await response.json();
-      } else {
-        result.error = await response.text();
-      }
-      console.log(result.data);
-      setEditFood(result.data);
+      queryAPI({
+        url: `http://localhost:5000/menu_items/${id}`,
+        onSuccess: (result) => toast.success(result.data),
```

Comment:
This is looking so sharp, great work!

Is it correct to assume that a future PR is coming that will add more functionality to this `onSuccess` callback?

Reply:
Yes